app.py

```diff
@@ -0,0 +1,8 @@
+from flask import Flask
+from flask_restful import Api
+
+app = Flask(__name__)
```

Comment:
I was thinking maybe now that the code base is not that big, it may not be a bad idea to switch to FastAPI as it provides Swagger out of the box; But I'm not sure about your deadlines, So you decide if you want to continue with Flask or FastAPI.